### PR TITLE
Add messages in README.md for onboarding tutorial

### PR DESCRIPTION
**Pull request by e-wai**
+ Made into the README.md to demonstrate the git flow for strategy

Better PR:
+ context on ticket 
+ information on "why" we approached the ticket the way we did
+ reviewers, check this change in x file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,3 +54,6 @@ Message Emma Wai or Michael Shiozaki on Slack.
 - https://www.codecademy.com/learn/learn-python
 - https://www.learnpython.org/
 - https://dabeaz-course.github.io/practical-python/
+
+Hello everyone :)
+: (
```
Comment (ufshk) [inline README.md:59]: Unnecessary and saddening comment

**Review by ufshk — CHANGES_REQUESTED** [review]
please remove this line thanks

File: README.md (hunk 1 above)
Comment (e-wai) [reply README.md:59]: It's in honour of the term starting

**Comment by e-wai** [comment]
Just an example

**Pull request closed without merging**